Weather/FlagoftheCharles.15m.rb

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/ruby
+
+# <bitbar.title>Flag of the Charles</bitbar.title>
+# <bitbar.version>v1.0</bitbar.version>
+# <bitbar.author>Isaac</bitbar.author>
+# <bitbar.author.github>irstacks</bitbar.author.github>
+# <bitbar.desc>Shows Boston's current Community Boating, Inc flag and related weather information.</bitbar.desc>
+# <bitbar.image>http://www.community-boating.org/wp-content/themes/communityboating/images/burgee.png</bitbar.image>
```

Comment:
This should be a screenshot of the plugin working. Could you amend this please 😄